tsconfig.spec.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What is the purpose of this file?

Reply:
it extends the Typescript config with unit test specific params